### PR TITLE
Update finch to v0.8.2

### PR DESCRIPTION
**Pull request by Zeitsperre**
## Overview

Please include a summary of the changes and which issues are fixed. 

Please also include relevant motivation and context. 

List any dependencies that are required for this change.

## Changes

- Finch: update `finch` component 
  from [0.7.7](https://github.com/bird-house/finch/releases/tag/v0.7.7)
  to [0.8.2](https://github.com/bird-house/finch/releases/tag/v0.8.2)

  Relevant Changes:
  - v0.8.0
    - Add hourly_to_daily process
    - Avoid annoying warnings by updating birdy (environment-docs)
    - Upgrade to clisops 0.8.0 to accelerate spatial averages over regions. 
    - Upgrade to xesmf 0.6.2 to fix spatial averaging bug not weighing correctly cells with varying areas. 
    - Update to PyWPS 4.5.1 to allow the creation of recursive directories for outputs.
  - v0.8.2
    - Add ``geoseries_to_netcdf`` process, converting a geojson (like a OGC-API request) to a CF-compliant netCDF. 
    - Add ``output_name`` argument to most processes (excepted subsetting and averaging processes), to control 
      the name (or prefix) of the output file. 
    - New dependency ``python-slugify`` to ensure filenames are safe and valid. 
    - Pinning dask to ``<=2022.1.0`` to avoid a performance issue with ``2022.1.1``.

## Related Issue / Discussion

- Resolves https://github.com/bird-house/finch/issues/230
- Needed https://github.com/Ouranosinc/pavics-sdi/pull/252

### CODE REVIEW

**Comment by Zeitsperre** [comment]
run tests

**Comment by tlvu** [comment]
run tests

**Comment by Zeitsperre** [comment]
@matprov Looks like there's an issue with the Jenkins at CRIM. Think you could investigate?
```
...
failed to register layer: Error processing tar file(exit status 1): write /opt/conda/pkgs/cache/18414ddb.json: no space left on device
...
```

**Comment by matprov** [comment]
run tests

**Comment by matprov** [comment]
Hi @Zeitsperre, it's fixed, thanks for highlighting this.

**Comment by tlvu** [comment]
@matprov can you add @Zeitsperre to the whitelist of people that can trigger the IaC pipeline.  Trevor will be taking care of keeping Finch and Raven up-to-date in the stack.  Thanks.

**Comment by matprov** [comment]
@Zeitsperre you should now have the right to trigger the pipeline via the PR. Please tell me if it works.

**Comment by Zeitsperre** [comment]
run tests

**Comment by Zeitsperre** [comment]
Thanks @matprov

**Comment by Zeitsperre** [comment]
run tests

**Comment by Zeitsperre** [comment]
Well that doesn't look good.

**Comment by matprov** [comment]
> Well that doesn't look good.

@Zeitsperre Do you know from where all the `ha:////4ItaOhsIRFfcNciVdA3A5uvEe7JeZm9h..` come from? First time I see this in the pipeline.

**Comment by Zeitsperre** [comment]
run tests

**Comment by tlvu** [comment]
@Zeitsperre Let's deploy this PR on your personal PAVICS server.  Can trigger tests faster than this pipeline that provision an entire new PAVICS stack each time and runs the full test suite on it, then destroy the stack so you can not re-run the tests again.

There's only `subset-user-input.ipynb` that keeps failing repeatedly so we do not have to re-run the entire test suite each time.

**Comment by tlvu** [comment]
To finish and merge this PR:
* update CHANGE.md, commit, push
* update PR description with more or less same info as CHANGE.md
* merge with `master` if needed
* run `bump2version patch`, push
* merge this PR, copying the PR description into the merge commit description 
* git tag the same version bump2version did on the merge commit

**Comment by Zeitsperre** [comment]
@tlvu This is good for review. Thanks!

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +16,34 @@
 
 [//]: # (list changes here, using '-' for each new entry, remove this when items are added)
 
+[1.18.9](https://github.com/bird-house/birdhouse-deploy/tree/1.18.9) (2022-03-16)
+------------------------------------------------------------------------------------------------------------------
+
+[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+
+[1.18.9](https://github.com/bird-house/birdhouse-deploy/tree/1.18.8) (2022-03-16)
+------------------------------------------------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (tlvu) [reply CHANGES.md]: LOL yeah you do not set the version number and the release date yourself, bump2version will do this automatically.

**Review by tlvu — APPROVED** [review]
+1 all good, congrats for first release :D

**Comment by tlvu** [comment]
@matprov FYI, there's something fishy with the IaC pipeline.

In https://daccs-jenkins.crim.ca/job/PAVICS-e2e-workflow-tests/job/master/733/consoleFull, `subset-user-input.ipynb` is failing with
```
21:14:56  ___ pavics-sdi-master/docs/source/notebooks/subset-user-input.ipynb::Cell 14 ___
21:14:56  Notebook cell execution failed
21:14:56  Cell 14: Cell execution caused an exception
21:14:56  
21:14:56  Input:
21:14:56  lon0 = float(bounds.minx)
21:14:56  lon1 = float(bounds.maxx)
21:14:56  lat0 = float(bounds.miny)
21:14:56  lat1 = float(bounds.maxy)
21:14:56  
21:14:56  result_tasmin = finch.subset_bbox(
21:14:56      resource=data,
21:14:56      variable="tasmin",
21:14:56      lon0=lon0,
21:14:56      lon1=lon1,
21:14:56      lat0=lat0,
21:14:56      lat1=lat1,
21:14:56      start_date="1958-01-01",
21:14:56      end_date="1958-12-31",
21:14:56  )
21:14:56  
21:14:56  Traceback:
21:14:56  
21:14:56  ---------------------------------------------------------------------------
21:14:56  ServiceException                          Traceback (most recent call last)
21:14:56  Input In [15], in <module>
21:14:56        3 lat0 = float(bounds.miny)
21:14:56        4 lat1 = float(bounds.maxy)
21:14:56  ----> 6 result_tasmin = finch.subset_bbox(
21:14:56        7     resource=data,
21:14:56        8     variable="tasmin",
21:14:56        9     lon0=lon0,
21:14:56       10     lon1=lon1,
21:14:56       11     lat0=lat0,
21:14:56       12     lat1=lat1,
21:14:56       13     start_date="1958-01-01",
21:14:56       14     end_date="1958-12-31",
21:14:56       15 )
21:14:56  
21:14:56  File </opt/conda/envs/birdy/lib/python3.8/site-packages/birdy/client/base.py-394>:5, in subset_bbox(self, resource, lon0, lon1, lat0, lat1, start_date, end_date, variable)
21:14:56  
21:14:56  File /opt/conda/envs/birdy/lib/python3.8/site-packages/birdy/client/base.py:368, in WPSClient._execute(self, pid, **kwargs)
21:14:56      365 mode = self._mode if self._processes[pid].storeSupported else SYNC
21:14:56      367 try:
21:14:56  --> 368     wps_response = self._wps.execute(
21:14:56      369         pid,
21:14:56      370         inputs=wps_inputs,
21:14:56      371         output=wps_outputs,
21:14:56      372         mode=mode,
21:14:56      373         lineage=self._lineage,
21:14:56      374     )
21:14:56      376     if self._interactive and self._processes[pid].statusSupported:
21:14:56      377         if self._notebook:
21:14:56  
21:14:56  File /opt/conda/envs/birdy/lib/python3.8/site-packages/owslib/wps.py:359, in WebProcessingService.execute(self, identifier, inputs, output, mode, lineage, request, response)
21:14:56      357 # submit the request to the live server
21:14:56      358 if response is None:
21:14:56  --> 359     response = execution.submitRequest(request)
21:14:56      360 else:
21:14:56      361     response = etree.fromstring(response)
21:14:56  
21:14:56  File /opt/conda/envs/birdy/lib/python3.8/site-packages/owslib/wps.py:921, in WPSExecution.submitRequest(self, request)
21:14:56      919 self.request = request
21:14:56      920 reader = WPSExecuteReader(verbose=self.verbose, timeout=self.timeout, auth=self.auth)
21:14:56  --> 921 response = reader.readFromUrl(
21:14:56      922     self.url, request, method='Post', headers=self.headers)
21:14:56      923 self.response = response
21:14:56      924 return response
21:14:56  
21:14:56  File /opt/conda/envs/birdy/lib/python3.8/site-packages/owslib/wps.py:602, in WPSExecuteReader.readFromUrl(self, url, data, method, username, password, headers, verify, cert)
21:14:56      595 def readFromUrl(self, url, data={}, method='Get', username=None, password=None,
21:14:56      596                 headers=None, verify=None, cert=None):
21:14:56      597     """
21:14:56      598     Reads a WPS status document from a remote service and returns the XML etree object.
21:14:56      599     :param str url: the URL to submit the GET/POST request to.
21:14:56      600     """
21:14:56  --> 602     return self._readFromUrl(url, data, self.timeout, method, username=username, password=password,
21:14:56      603                              headers=headers, verify=verify, cert=cert)
21:14:56  
21:14:56  File /opt/conda/envs/birdy/lib/python3.8/site-packages/owslib/wps.py:513, in WPSReader._readFromUrl(self, url, data, timeout, method, username, password, headers, verify, cert)
21:14:56      510     return etree.fromstring(u.read())
21:14:56      512 elif method == 'Post':
21:14:56  --> 513     u = openURL(url, data, method='Post',
21:14:56      514                 username=self.auth.username, password=self.auth.password,
21:14:56      515                 headers=headers, verify=self.auth.verify, cert=self.auth.cert, timeout=timeout)
21:14:56      516     return etree.fromstring(u.read())
21:14:56      518 else:
21:14:56  
21:14:56  File /opt/conda/envs/birdy/lib/python3.8/site-packages/owslib/util.py:211, in openURL(url_base, data, method, cookies, username, password, timeout, headers, verify, cert, auth)
21:14:56      208 req = requests.request(method.upper(), url_base, headers=headers, **rkwargs)
21:14:56      210 if req.status_code in [400, 401]:
21:14:56  --> 211     raise ServiceException(req.text)
21:14:56      213 if req.status_code in [404, 500, 502, 503, 504]:    # add more if needed
21:14:56      214     req.raise_for_status()
21:14:56  
21:14:56  ServiceException: <?xml version="1.0" encoding="utf-8"?>
21:14:56  <ExceptionReport version="1.0.0"
21:14:56      xmlns="http://www.opengis.net/ows/1.1"
21:14:56      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
21:14:56      xsi:schemaLocation="http://www.opengis.net/ows/1.1 http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd">
21:14:56      <Exception exceptionCode="NoApplicableCode" locator="NotAcceptable">
21:14:56          <ExceptionText>Request failed: (&#x27;Connection aborted.&#x27;, RemoteDisconnected(&#x27;Remote end closed connection without response&#x27;))</ExceptionText>
21:14:56      </Exception>
21:14:56  </ExceptionReport>
21:14:56  
```

Whereas here is a build on your Jenkins, started against a PAVICS VM on our side, with this PR, and the same `subset-user-input.ipynb` passed: https://daccs-jenkins.crim.ca/job/PAVICS-e2e-workflow-tests/job/master/734/console

**Pull request merged**